libcaf_net/caf/net/http/v1.cpp

```diff
@@ -32,8 +36,8 @@ writer& operator<<(writer& out, const std::string& str) {
 
 } // namespace
 
-std::pair<std::string_view, byte_span> split_header(byte_span bytes) {
-  std::array<std::byte, 4> end_of_header{{
+std::pair<std::string_view, byte_span> split_header(const byte_span bytes) {
```

Comment:
Adding `const` here makes sense to me. You can still override all of the bytes pointed to. This basically only prevents re-assigning the span variable itself.

```suggestion
std::pair<std::string_view, byte_span> split_header(byte_span bytes) {
```

Reply:
The first question I got reading the name `split_header` was does the function modify the range pointed to by bytes, and the fastest way to check that was adding `const`. I left it there since I like the message it conveys through the function signature. However, I don't mind leaving the old signature.